fiftyone/utils/voc.py

```diff
@@ -783,13 +803,16 @@ def _parse_attribute(value):
     except:
         pass
 
-    if value in {"True", "true"}:
-        return True
+    try:
```

Comment:
why would these checks every fail?

Reply:
The attributes are loaded in as a dictionary like the example in the PR description, so this call sometimes passes in a `dict`:
https://github.com/voxel51/fiftyone/blob/f12d3dc463e79b3b518f369a6e7892bbc9a7acb0/fiftyone/utils/voc.py#L529-L532

Reply:
I moved the cvat attribute parsing so that this isn't an issue.